src/plugins/hexcolor/README.md

```diff
@@ -0,0 +1,54 @@
+- infos = Information about the hexcolor plugin is in keys below
+- infos/author = Philipp Gackstatter <philipp.gackstatter@student.tuwien.ac.at>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides =
+- infos/recommends =
+- infos/placements = presetstorage
+- infos/status = maintained unittest nodep
+- infos/metadata = check/hexcolor
+- infos/description = Validation of hexcolors
+
+## Introduction
+
+Copy this hexcolor if you want to start a new
+plugin written in C.
+
+## Usage
+
+- add your plugin in `src/plugins/README.md`
+- and rewrite the rest of this `README.md` to give a great
+  explanation of what your plugin does
+
+## Dependencies
+
+None.
+
+## Examples
+
+```sh
+sudo kdb mount /tests/color.ini /tests/ipaddr ini hexcolor
+#> Mount a config file with the hexcolor plugin
+
+kdb setmeta /tests/color/hex check/hexcolor any
+#> Check the /tests/color/hex key for validity
+
+kdb set /tests/color/hex "#fff"
+#> Suceeds, since the value is a valid hexcolor. Quotes are important!
```

Comment:
This is not valid for shell recorder, see tests/shell/shell_recorder/tutorial_wrapper/README.md